microsoftgraph/microsoft-graph-explorer
# Add msaljs scopes

## Overview

- Changes the login implementation from loginRedirect to loginPopup
- Decodes the scopes from the access token
- Allows scopes modification using MSALjs instead of hellojs
- Removes the display of an 'admin consent required' alert when selecting scopes


### Demo

N/A

### Notes

N/A

## Testing Instructions

* Log in with an admin user account
* Select modify permissions
* Choose a new set of scopes and click modify permissions
* On completion, check the new list of scopes . You should find the list includes the list of scopes selected
